starfyre/__main__.py

```diff
@@ -14,43 +16,69 @@ def write_js_file(path):
     js_store = pkg_resources.path("starfyre.js", "store.js")
     shutil.copy(str(js_store), path + "/dist/store.js")
 
-
-def create_main_file(path):
-    """
-    Creates a main file in the build directory.
-    
-    This file will be used to run the project.
-    Every starfyre build will have an `__init__.py` file in the build directory.
-    And the `__init__.py` file will have a component that will render the root component. It will be named `app`.
-
-    You can have a look at the `test-application/build/__init__.py` file to see what it looks like.
-
-    The main file is also responsible for adding the `store.js` file to the `index.html` file.
-    """
+def build_routes_output(generated_routes, path):
     output_file_path = path + "/build/__main__.py"
     write_js_file(path)
 
     with open(output_file_path, "w") as f:
         f.write(
-            """
-from . import app
+f"""
+from starfyre import create_component, render_root
+from starfyre.exceptions import IndexFileConflictError
+
 import os
+import sys
 from pathlib import Path
+import importlib
+
+def _build_output(generated_routes, path):
```

Comment:
Two suggestions - 
- Can we use a different name here? Something like `generate_pages` or `create_pages` or on these lines?

- Can you please add a docstring?

Reply:
And I suppose this logic will be changed after our discussion today.